functions/execution-context-and-closures.md

```diff
@@ -0,0 +1,56 @@
+# Execution Context and Closures
+
+## Goals
+
+Our goals for this lesson is to introduce at a high-level how nested functions can share context variables in a particular scope, known as a "closure." By doing so, we will have a better understanding of what makes JavaScript functions and callback functions so powerful as we gear up for consuming APIs.
+
+## Introduction
+
+Before diving into closures, however, we need to begin with a little context. _Execution_ context, that is!  
+
+The **execution context** describes the current environment within which JavaScript code is being run. By its very name, this environment can change depending on when the lines of code are run or where the lines of code are scoped. It's all about the _context_ of when and where JavaScript code is _executed_, which will define the scope of variables and function declarations.  
```

Comment:
I suggest we build a vocab block like in earlier lessons for:

- Closure
- Execution Context
- Global Execution Context